static/app/views/monitors/components/monitorCheckIns.tsx

```diff
@@ -168,7 +169,11 @@ function MonitorCheckIns({monitor, monitorEnvs, orgSlug}: Props) {
                               avatarSize={12}
                             />
                           }
-                          to={`/issues/${id}`}
+                          to={
+                            USING_CUSTOMER_DOMAIN
+                              ? `/issues/${id}`
+                              : `/organizations/${organization.slug}/issues/${id}`
+                          }
```

Comment:
link component will handle removing the organization + slug for you but you need to start with the full link
```suggestion
                          to={`/organizations/${organization.slug}/issues/${id}/`}
```